tests/view/tables/mock_tables.py

```diff
@@ -0,0 +1,20 @@
+class TableMock:
+    """Table mock"""
+
+    query = None
+
+    def __init__(self, **kwargs):
+        self.tables = kwargs.get("tables", [])
```

Comment:
@paulodamaso Why do you initialize ``self.tables`` as well? The way I understand it this class should be used as follows:
``TableMock.query.tables``, so does it make sense to add the tables to ``self`` as well? I think only ``self.query`` is enough.

Reply:
@amihaiemil `TableMock` should reflect `Table` behavior, so I designed it so we could use it as an mocked database of tables. I feel that it is more logical this way, having the table objects to belong to table than query.

Reply:
@paulodamaso I see, ok

Reply:
@paulodamaso This is the same of #375 right? No mock objects